src/eventsource.js

```diff
@@ -37,15 +37,15 @@
 /**
  * @typedef {Object} Event
  * @memberof OpenSeadragon
- * @param {boolean} [stopPropagation=undefined] - If set to true, the event exits after handling the current call.
+ * @property {boolean} [stopPropagation=undefined] - If set to true, the event exits after handling the current call.
  */
 
 /**
  * Event handler method signature used by all OpenSeadragon events.
  *
  * @typedef {function(OpenSeadragon.Event): void} OpenSeadragon.EventHandler
  * @memberof OpenSeadragon
- * @param {OpenSeadragon.Event} event - The event object containing event-specific properties.
+ * @property {OpenSeadragon.Event} event - The event object containing event-specific properties.
```

Comment:
For instance this. Although it's a type definition, it's a type definition for a function, so I believe that means this should be `@param`.